### PR TITLE
ref(new-widget-builder-experience): Update text copy

### PR DESCRIPTION
**Pull request by narsaynorath**
We've received some feedback to update some of our text copy. It's mostly what's covered in the notion doc, but since there's more documentation about the data set selector, I've linked it: https://docs.sentry.io/product/dashboards/custom-dashboards/#data-set-selection

### CODE REVIEW

File: static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx [hunk 1]
```diff
@@ -49,20 +49,20 @@ export function ColumnsStep({
...
               {
-                tagFieldLink: (
+                fieldTagLink: (
                   <ExternalLink href="https://docs.sentry.io/product/sentry-basics/search/searchable-properties/#event-properties" />
                 ),
               }
             )
           : tct(
-              'To group events, add [functionLink: functions] f(x) that may take in additional parameters. [tagFieldLink: Tag and field] columns will help you view more details about the events (i.e. title).',
+              'To stack events, add [functionLink: functions] f(x) that may take in additional parameters. [fieldTagLink: Field and tag] columns will help you view more details about the events (e.g., title).',
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx:60]: nit: we are using the 'group' wording in a couple of places. shouldn't we stick to it? in the notion's doc is also group and in the sentry doc we have a phrase with `Bar charts will group results by day`

File: static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx [hunk 2]
```diff
@@ -48,8 +49,13 @@ export function DataSetStep({
   return (
     <BuildStep
       title={t('Choose your data set')}
-      description={t(
-        'This reflects the type of information you want to use. For a full list, read the docs.'
+      description={tct(
+        `This reflects the type of information you want to use. To learn more, [link: read the docs].`,
+        {
+          link: (
+            <ExternalLink href="https://docs.sentry.io/product/dashboards/custom-dashboards/#data-set-selection" />
```
Comment (priscilawebdev) [inline static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx:56]: 💜

**Review by priscilawebdev — APPROVED** [review]
looks good to me. thanks for adding these changes! 🎉

File: static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx (hunk 1 above)
Comment (jas-kas) [reply static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx:60]: I actually took from our documentation ~
See here: [](https://docs.sentry.io/product/discover-queries/query-builder/#stacking-functions)
When speaking of functions, we call that "stacking functions" that allow events to be "stacked"

File: static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx [hunk 3]
```diff
@@ -11,7 +12,7 @@ import {BuildStep} from './buildStep';
 
 const DATASET_CHOICES: [DataSet, string][] = [
   [DataSet.EVENTS, t('Events (Errors, transactions)')],
```
Comment (jas-kas) [inline static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx:14]: Part of the copy updates was changing this data set name. Is this reflected anywhere in the PR?

Comment (narsaynorath) [reply static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx:14]: Ah, I remembered seeing it but I noticed in the Notion doc it was removed by you (probably an accident) and was going to follow up. I'll update it in this PR, "Errors and Transactions" is what we agreed on, correct?

Comment (narsaynorath) [reply static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx]: https://github.com/getsentry/sentry/pull/33530/files#diff-7bbe835240512c55ba82968c38e19c27d0366c86831d3304874bf80bb57612e6R14

The change is up now :+1:

Comment (jas-kas) [reply static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx]: Def an accident. Thanks Nar!

**Pull request merged**